### ISSUE TITLE
Unable to use LDAP authentification 

### ISSUE THREAD

**Issue by jeeberhardt** [uhm-coe/authorizer#2]
Hi !
I try to use the LDAP authentification to allow users of my institute to have an access to the website.
But it doesn't work (but Google logins does) ... and I don't know why.

```
Wordpress == 4.3.1
Authorizer == 2.3.3
```

Here the LDAP configuration:
```
LDAP Host: igbmc.u-strasbg.fr
LDAP Search base: ou=utilisateurs,dc=igbmc,dc=u-strasbg,dc=fr
```

And I know my credentials and the LDAP informations (host and dn) are corrects because I can access to the LDAP server through python (using python-ldap). Activate/deactivate TLS doesn't change anything. And I don't know why the LDAP Directory User Password is constantly filled.

Here the current configuration of Authorizer: https://imgur.com/8WqWU3T

Do you have an idea ? Thank you ! :-)

EDIT: And can you tell me where the LDAP informations are stored for Authorizer ?

EDIT: with the plugin ```Active Directory Integration``` (https://wordpress.org/plugins/active-directory-integration/), it works.

**Comment by figureone** [comment]
Hi, sorry for the delay. Authorizer isn't currently configured to allow anonymous binds, so you need to fill out the "LDAP Directory User" and "LDAP Directory User Password" fields.
https://github.com/uhm-coe/authorizer/blob/master/authorizer.php#L872

I can work on an update that allows anonymous binds, but it will take a bit of time. Let me know!

Finally, configuration is stored in the wp_options table; most of the config (including LDAP config) is saved as a serialized PHP array in the auth_settings option.

**Comment by figureone** [comment]
I just released version 2.3.6 which should allow anonymous binds.
https://wordpress.org/plugins/authorizer/changelog/
